python/src/aiconfig/editor/client/src/components/prompt/prompt_input/schema_renderer/PromptInputDataSchemaRenderer.tsx

```diff
@@ -14,12 +15,15 @@ export default memo(function PromptInputDataSchemaRenderer({
   data,
   onChangeData,
 }: Props) {
+  const {readOnly} = useContext(AIConfigContext);
+  
   switch (schema.type) {
     case "string":
       return (
         <Textarea
           value={data ? (data as string) : ""}
           onChange={(e) => onChangeData(e.target.value)}
+          disabled={readOnly}
```

Comment:
Same here r.e. using the TextRenderer w/ Spoiler

Reply:
updated